src/components/contextual/pages/pool/PoolCompositionCard/components/TokenBreakdown.vue

```diff
@@ -122,3 +130,23 @@ const shareOfTokenInPool = computed((): number => {
     />
   </template>
 </template>
+<style scoped>
+.nested-token {
+  @apply flex-shrink-0 mr-2 relative ml-1 sm:ml-0;
+}
+
+.nested-token::after {
+  content: '';
+  top: 14px;
```

Comment:
Needed custom pixel perfect values here for the tree structure to line up perfectly.